### PR TITLE
inkscape: Fix extract_dir and change the version number

### LINKED ISSUE #116: inkscape : extraction do not work
manifest : inkscape
version : 1.3.2_2023-11-25_091e20e

The extract_dir is trying to extract a folder named "inkscape-1.3.2_2023-11-25_091e20e", which does not exist. It should extract the folder "inkscape" instead

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)
- [ ] ✨ Add Manifest
- [x] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other
## Description
Fix the extract_dir to use "inkscape" and change the version number the remove the date and commit hash
## Related Tickets & Documents
Fixes #116
## [optional] What gif/emoji best describes this PR or how it makes you feel?
🚑️🚨
### Add/Modify a description in the manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Check URLs?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update to the latest version?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Update Hashes?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Format the json manifest?
- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed
### Notes in the manifest?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Dependencies?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Application Suggestions?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Pre-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### Post-installation steps to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed
### File persistence to perform?
- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### inkscape

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87 — APPROVED** [review]

**Pull request merged**
